client/core/types.go

```diff
@@ -397,3 +397,18 @@ func coinIDString(assetID uint32, coinID []byte) string {
 	}
 	return coinStr
 }
+
+// InitDEXConnsResult holds data returned from initializeDEXConnections
+type InitDEXConnsResult struct {
```

Comment:
That was actually close to the first name I had for it `DEXLiteResults` because really it's a lite version of what you can get from the `exchanges` endpoint.